### ISSUE TITLE
auth issue: KeyError: 'nickname'

### ISSUE THREAD

**Issue by nirik** [fedora-infra/bodhi#4565]
I authenticated with new bodhi and got a token, but now when I try and do things from the cli I get: 

```
2022-06-13 19:19:31,797 ERROR [bodhi.server][ThreadPoolExecutor-0_1] Error caught.  Handling JSON response.
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/pyramid/tweens.py", line 41, in excview_tween
    response = handler(request)
  File "/usr/lib/python3.10/site-packages/pyramid/router.py", line 147, in handle_request
    response = _call_view(
  File "/usr/lib/python3.10/site-packages/pyramid/view.py", line 683, in _call_view
    response = view_callable(context, request)
  File "/usr/lib/python3.10/site-packages/pyramid/viewderivers.py", line 401, in viewresult_to_response
    result = view(context, request)
  File "/usr/lib/python3.10/site-packages/pyramid/viewderivers.py", line 144, in _requestonly_view
    response = view(request)
  File "/usr/lib/python3.10/site-packages/bodhi/server/auth/views.py", line 114, in login_with_token
    get_and_store_user(request, token["access_token"], response)
  File "/usr/lib/python3.10/site-packages/bodhi/server/auth/utils.py", line 39, in get_and_store_user
    username = userinfo['nickname']
KeyError: 'nickname'
```

**Comment by mattiaverga** [comment]
Looks like #4522 
cc @abompard

**Comment by abompard** [comment]
Fixed by #4570, thanks
